code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -1269,3 +1270,49 @@
 	icon_state = "overall_skirt_x"
 	item_state = "overall_skirt_x"
 /********** Overalls End **********/
+
+// Pronoun pins
+/obj/item/clothing/accessory/pronoun
+	name = "any/all pronouns pin"
+	desc = "A pin denoting the wearer's pronouns: any/all."
+	icon = 'icons/clothing/accessories/pronoun_pin.dmi'
+	icon_state = "pronounpin"
+	item_state = "pronounpin"
+	worn_overlay = "over"
+	drop_sound = 'sound/items/drop/card.ogg'
+	pickup_sound = 'sound/items/pickup/card.ogg'
+	contained_sprite = TRUE
+	build_from_parts = TRUE
+	overlay_in_inventory = FALSE
+
+/obj/item/clothing/accessory/pronoun/hehim
+	name = "he/him pronouns pin"
+	desc = "A pin denoting the wearer's pronouns: any/all."
+
+/obj/item/clothing/accessory/pronoun/hehim
```

Comment:
you've got the he/him pin twice

Reply:
Just noticed this myself, oops. Fixed these with https://github.com/Aurorastation/Aurora.3/pull/16446/commits/9a6aac672b40e32102b6849d156e00dae857c1e9